portal-web/test/functional/com/liferay/portalweb/macros/BalloonEditor.macro

```diff
@@ -0,0 +1,33 @@
+definition {
+
+	macro addTable {
+		AssertElementPresent(locator1 = "CKEditorBalloonEditorSample#INSERT_BTN");
```

Comment:
I recommend to remove this Assertion line. Click already has an inbuilt `waitForVisible`, if for some reason it's not "visible" then there's also `Click.clickAtNotVisible` too which has inbuilt `waitForElementPresent`.

We want to try to use `verifyElementPresent` or `waitForElementPresent` instead of assertions for macros. Reason is they'll likely be re-used in other tests and that assertion isn't the main point of the test.